lib/MediaInfo.js

```diff
@@ -379,10 +379,10 @@ class MediaInfo {
 				for (let codec of this.codecs.values())
 				{
 					//If that codec is supported
-					if (supportedCodecs.has(codec.getCodec().toLowerCase()))
+					if (supportedCodecs.has(codec.getType()))
```

Comment:
codec matching in SDP answer is done by checking the codec name, not by payload type. 

Currently we have a limitation that we only allow a single codec with the same name in the response, which we need to improve.  Maybe instead of looking for the key of the map, we need to iterate the values of the map and look for the elements with the matching codec name.

Maybe we have to

Reply:
right, reverting this change then